src/Catty UITests/ScriptCollectionView/BrickCellPickerViewerTest.swift

```diff
@@ -151,25 +126,6 @@ class BrickCellPickerViewerTests: XCTestCase {
             app.pickerWheels.element.adjust(toPickerWheelValue: testValues[2])
             app.buttons[kLocalizedDone].tap()
             XCTAssert(waitForElementToAppear(app.otherElements[ACCESSABILITY_LABEL_MESSAGE + testValues[2]]).exists, "Error while changing variable for Brick: " + brick)
```

Comment:
Here we have the same case.
Does this really test the functionality it is intended for? ;)

Reply:
I think here it works fine since after the done button is pressed it checks if the last message entered `MessageView_testVariable3` exists or not. And after the cancel button is pressed it checks if the previous message exists or not?

please correct me if I am missing something.

Reply:
Ideally it should work just like the test changeVariable. But since there is a bug in these control bricks, which just leaves the last variable that you created (and deletes the other elements of the list), you can't test the other values.
So we have to keep that in mind.

Reply:
I understand what you are saying but I think the test will work fine after the bug is removed since the test uses `app.pickerWheels.firstMatch.swipeDown()` instead of `app.pickerWheels.element.adjust(toPickerWheelValue: testValues[2])` to select the control to test the cancel button.